bin/update

```diff
@@ -33,6 +38,9 @@ Dir.chdir APP_ROOT do
     execute "bin/rake evm:automate:reset"
   end
 
+  # Make sure bower is done before compiling assets
+  bower_thread.join
```

Comment:
@himdel is this right?  we need to join in the event bower is still running so we don't compile assets while it's still installing javascript stuff, right?